src/js/view/controls/controlbar.js

```diff
@@ -163,6 +170,11 @@ export default class Controlbar {
         }, localization.liveBroadcast);
         liveButton.element().textContent = localization.liveBroadcast;
 
+        const fullscreenButton = button('jw-icon-fullscreen', () => {
```

Comment:
Your `changeFullscreenTooltip` function is in the outer scope, therefore you're passing it variables that would be accessible if it were in the inner scope. Move the function in the inner scope and remove the unnecessary variables (ie. instead of passing a model and getting its localization, you would already have the localization var readily available). Less variable passing and declarations makes the codebase smaller.

Reply:
in fact no sense in having a separate `changeFullscreenTooltip` since it only gets called in one place